packages/teleport/src/cluster/components/Support/Support.tsx

```diff
@@ -0,0 +1,172 @@
+            />
+          </MiniBox>
+          <MiniBox>
+            <Header title="Resources" icon={<Icons.Stars />} />
+            <SupportLink
+              title="Quickstart Guide"
+              url="https://gravitational.com/teleport/docs/quickstart/"
+            />
+            <SupportLink
+              title="tsh User Guide"
+              url="https://gravitational.com/teleport/docs/user-manual/"
+            />
+            <SupportLink
+              title="Admin Guide"
+              url="https://gravitational.com/teleport/docs/admin-guide/"
+            />
+            <SupportLink
+              title="Download Page"
+              url="https://gravitational.com/teleport/download"
+            />
+            <SupportLink
+              title="FAQ"
+              url="https://gravitational.com/teleport/docs/faq/"
+            />
+          </MiniBox>
+          <MiniBox>
+            <Header title="Troubleshooting" icon={<Icons.Stars />} />
+            <SupportLink title="Monitoring Teleport" url="" />
+            <SupportLink title="Collecting Debug Data" url="" />
+            <SupportLink title="Troubleshooting FAQ" url="" />
+          </MiniBox>
+          <MiniBox>
+            <Header title="Updates" icon={<Icons.Stars />} />
+            <SupportLink
+              title="Product Changelog"
+              url="https://github.com/gravitational/teleport/blob/master/CHANGELOG.md"
+            />
+            <SupportLink
+              title="Gravitational Blog"
+              url="https://gravitational.com/blog/"
+            />
+          </MiniBox>
+        </Flex>
+      </Card>
+      <ClusterBox
+        border="1px solid"
+        borderColor="primary.light"
+        mt={8}
+        mb={10}
+        borderRadius={3}
+        p={5}
+      >
+        <Text as="h5" typography="h5" mb={4} fontWeight="bold">
+          Cluster Information
+        </Text>
+        <ClusterData title="Cluster Name" data={cluster.clusterId} />
+        <ClusterData title="Teleport Version" data={cluster.authVersion} />
+        <ClusterData title="Public Address" data={cluster.publicURL} />
+        <ClusterData title="Connected Nodes" data={cluster.nodeCount} />
+      </ClusterBox>
+    </FeatureBox>
+  );
+};
+
+const SupportLink = ({ title = '', url = '' }) => (
+  <Text mb={3}>
+    <Link href={url} target="_blank" mb={2}>
+      {title}
+    </Link>
+  </Text>
+);
+
+const ClusterData = ({ title = '', data = null }) => (
+  <Flex mb={3}>
+    <Box width={130}>{title}: </Box>
+    {data}
+  </Flex>
+);
+
+const Header = ({ title = '', icon = null }) => (
+  <Box
+    borderBottom="1px solid"
+    borderColor="primary.dark"
+    mb={3}
+    width={210}
+    mt={4}
+    pb={2}
+  >
+    <Flex alignItems="center">
+      <Text pr={2} fontSize={30}>
+        {icon}
+      </Text>
+      <Text as="h5" typography="h5" fontWeight="bold">
+        {title}
+      </Text>
+    </Flex>
+  </Box>
+);
+
+const MiniBox = styled(Box)`
+  h5 {
```

Comment:
for simple tweaks you can use style properly directly 

```jsx
 <Link href={url} target="_blank" mb={2} style={{  "textDecoration": "none", "color": "inherit"  }} >
```

Reply:
revised